pkg/KubeArmorController/api/v1/kubearmorhostpolicy_types.go

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022 Authors of KubeArmor
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// KubeArmorHostPolicySpec defines the desired state of KubeArmorHostPolicy
+type KubeArmorHostPolicySpec struct {
+	NodeSelector NodeSelectorType `json:"nodeSelector"`
+
+	Process      ProcessType          `json:"process,omitempty"`
```

Comment:
hsp doesn't support the global allow policy.
this is the big difference between hsp and ksp. did you consider this point?

Reply:
I am not aware of this detail. However, the original host policy controller have the same structure. 
https://github.com/kubearmor/KubeArmor/blob/main/pkg/KubeArmorHostPolicy/api/security.kubearmor.com/v1/kubearmorhostpolicy_types.go#L240-L258